samples/win32.mak

```diff
@@ -45,18 +45,18 @@ wc2:
 	.\wc2.exe wc2.d
 
 winsamp:
-	$(DMD) winsamp $(DFLAGS) gdi32.lib user32.lib winsamp.def
+	$(DMD) winsamp $(DFLAGS) gdi32.lib user32.lib kernel32.lib winsamp.def
```

Comment:
Shouldn't be necessary

Reply:
You're right, but I added them cause I was getting a failure so I assumed `-defaultlib=` was getting set somewhere...maybe in the `DFLAGS` environment variable or something.  I'll remove these changes and see what happens.

Reply:
I had the logic swapped, it was only adding kernel32 if `-defaultlib=`. Fixed.